src/ast/irbuilderbpf.cpp

```diff
@@ -2170,7 +2172,7 @@ void IRBuilderBPF::CreateRingbufOutput(Value *data,
 
   SetInsertPoint(loss_block);
   CreateAtomicIncCounter(to_string(MapType::EventLossCounter),
-                         bpftrace_.event_loss_cnt_key_);
+                         bpftrace::BPFtrace::event_loss_cnt_key_);
```

Comment:
This kinda freaks me out. Should this member var not be static?

Reply:
It's the key value and value value (heh) for the map it writes into for the loss counter.

The impl should probably be refactored to use global vars instead of an arraymap.

Reply:
No, I meant that we're using a static class variable to read and increment.

Reply:
Oh sorry, I was thrown off by the IR instruction: `CreateAtomicIncCounter`